fetch.bs

```diff
@@ -54,10 +54,29 @@ urlPrefix:https://tc39.es/ecma262/#;type:dfn;spec:ecma-262
     url:realm;text:realm
     url:sec-list-and-record-specification-type;text:Record
     url:current-realm;text:current realm
+
+urlPrefix:https://www.ietf.org/archive/id/draft-annevk-johannhof-httpbis-cookies-00.html#;type:dfn;spec:cookies
```

Comment:
https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-layered-cookies

(for drafts we typically use datatracker so we don't have to update all links for each revision)